### PR TITLE
Added linter and few small fixes

### PR DESCRIPTION
**Pull request by Piterden**

### CODE REVIEW

**Comment by YouTwitFaceTG** [comment]
I have a eslint config that I use and I'd rather not change for this.

**Comment by Piterden** [comment]
Okay. But `Telegraf` should start with uppercased letter

**Comment by YouTwitFaceTG** [comment]
@Piterden I prefer it with a lowercase `T`

File: index.js [hunk 1]
```diff
@@ -30,11 +30,10 @@ bot.on(`new_chat_members`, async (ctx) => {
     const { first_name, id } = ctx.message.new_chat_member;
     const { title } = ctx.chat;
 
-    if (id == botId) {
+    if (id === Number(botId)) {
         const { user, status } = await ctx.getChatMember(ctx.from.id);
-        const statuses = [`creator`, `administrator`];
```
Comment (YouTwitFaceTG) [inline index.js]: I added this for clearness

**Pull request closed without merging**